MetadataExtractor/Extractors/Data/DicomExtract.py

```diff
@@ -0,0 +1,57 @@
+from .IDataExtract import IDataExtract
+from MetadataExtractor.Util import metadataCreation, metadataFormatter
+import pydicom
```

Comment:
please add pydicom to the [requirements.txt](https://github.com/BenediktHeinrichs/metadataextractor/blob/main/requirements.txt)

Reply:
This has not been resolved